content/en/docs/ops/deployment/requirements/index.md

```diff
@@ -109,6 +109,7 @@ The following ports and protocols are used by the Istio control plane (istiod).
 | 8080 | HTTP | Debug interface (deprecated) | No |
 | 443 | HTTPS | Webhooks | No |
 | 15014 | HTTP | Control plane monitoring | No |
+| 15053 | DNS | DNS resolution for Service Entries | No |
```

Comment:
This is a legacy of an experiment that wasn't fully cleaned up, its only in the yaml manifest not the code. we should fully clean it up there ,not document it.

Reply:
Or document that its in the agent (table just above this)

Reply:
Ah, so this was initially the design which is replaced/postceded by the local DNS Agent.

I'll move it to the table above as suggested.